src/main/java/Duke.java

```diff
@@ -1,10 +1,37 @@
-public class Duke {
+import java.util.Scanner;
+import java.util.Arrays;
+import java.util.ArrayList;
+public class Duke{
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String dash = "__________________________________";
+        String greet = dash + "\nHello I'm Duke\nWhat can I do for you?\n" + dash;
```

Comment:
Can possibly extract most of the methods to print user output to a separate class/method for better readability